frontend/package.json

```diff
@@ -17,10 +17,10 @@
     "@iconify/icons-mdi": "^1.2.9",
     "@iconify/react": "^3.2.1",
     "@monaco-editor/react": "^4.3.1",
-    "@mui/icons-material": "^5.16.7",
+    "@mui/icons-material": "^5.17.1",
```

Comment:
I wonder why the dependencies are changed here?

Are they required to get this to work? If not, can you please remove the dependency changes from the PR?